tests/conftest.py

```diff
@@ -48,17 +50,38 @@ def seed_application_records(
             app["fund_id"] = random_fund_id
             app["round_id"] = random_round_id
         app_flags = []
+        app_flags_v2 = []
         if "flags" in app:
             app_flags = app["flags"]
             app.pop("flags")
+        app_flags_v2 = []
+        if "flags_v2" in app:
+            app_flags_v2 = app["flags_v2"]
+            app.pop("flags_v2")
         inserted_application = bulk_insert_application_record(
             [app], "COF", True
         )[0]
         app["flags"] = app_flags
+        app["flags_v2"] = app_flags_v2
```

Comment:
Wait I'm confused, why did we `pop` it if we re-add it? 😅

Is it so it doesn't exist when calling `bulk_insert_application_record`?

Oh looking now you've just copied the v1 flag stuff

Reply:
Yeah - we don't want these extra bits of config that we include for seeding test data to get passed to `bulk_insert_application_record` so we take them out and then re-add.